build.py

```diff
@@ -44,33 +44,43 @@ def build_mac(release):
         return False
     return True
 
+def check_pkgconfig(lib, min_version):
+    try:
+        subprocess.check_output(
+            ['pkg-config', '--exists', '%s >= %s' % (lib, min_version)])
```

Comment:
Do you mean `pkg-config %s --atleast-version=%s`? `--exists` seem to be wrong.19.0.13 is 2.7.1 or 2.7.0? I have 2.8 and it says 20.0.14. Harfbuzz only needs to be 1.4.0, I think?